### PR TITLE
Corrected `raise Error as e` syntax

### PR DESCRIPTION
**Pull request by alongd**
Corrected all instances of `raise Error, e:` to the correct syntax `raise Error as e:`. The former won't be supported in Py3. 
Also, if `e` isn't used, it was deleted.
See https://www.python.org/dev/peps/pep-0008/#id51

### CODE REVIEW

File: rmgpy/rmg/model.py [hunk 1]
```diff
@@ -354,7 +354,7 @@ def makeNewSpecies(self, object, label='', reactive=True, checkForExisting=True)
         try:
             spec = Species(index=speciesIndex, label=label, molecule=[molecule], reactive=reactive,
                  thermo=object.thermo, transportData=object.transportData)
-        except AttributeError, e:
+        except AttributeErro:
```
Comment (mliu49) [inline rmgpy/rmg/model.py]: Letter missing.

File: rmgpy/rmg/model.py [hunk 2]
```diff
@@ -1952,7 +1952,7 @@ def getSpecies(self, obj):
             try:
                 spc = self.indexSpeciesDict[obj]
                 return spc
-            except KeyError, e:
+            except KeyError as e:
```
Comment (mliu49) [inline rmgpy/rmg/model.py]: This seems like an unnecessary try/except clause...

**Comment by alongd** [comment]
OK, let me know if that's OK and I'll squash

**Comment by mliu49** [comment]
Thanks! The fixups look good.

**Comment by alongd** [comment]
Should be good to go

**Pull request merged**